supporting_content/metrics/listunitprice.md

```diff
@@ -0,0 +1,77 @@
+
+### Microsoft
+
+- [Understand usage details fields - Microsoft Cost Management](https://learn.microsoft.com/en-us/azure/cost-management-billing/automate/understand-usage-details-fields)
+- [Azure Retail Prices REST API overview | Microsoft Learn](https://learn.microsoft.com/en-us/rest/api/cost-management/retail-prices/azure-retail-prices)
+
+### OCI
+
+- [Cost and Usage Reports Overview](https://docs.oracle.com/en-us/iaas/Content/Billing/Concepts/usagereportsoverview.htm)
+- [Estimate Your Monthly Cost (List Pricing REST API)](https://docs.oracle.com/en-us/iaas/Content/GSG/Tasks/signingup_topic-Estimating_Costs.htm#accessing_list_pricing)
+
+## Example usage scenarios
+
+- See [Appendix: Quantity-Driven Pricing strategies](../apendix/quantity_driven_pricing_strategies.md) section for more information on
+  - Quantity-Driven Pricing strategies (such as flat-rate, volume-based rates and tier-based rating)
+  - related UC scenarios
+  - Current values observed in billing data for various scenarios
+  - Alternative data sources for various scenarios
+- See [Pricing Support – UCs and Data samples Spreadsheet](https://docs.google.com/spreadsheets/d/1AZ-vtkKeKwYc8rqhxP1zMTnAVAS-svmWQQmr8cpv-IM/edit#gid=117987709) for additional UC scenarios and sample data.
+
+## Discussion / Scratch space
+
+### Current data sources
+
+- While Microsoft and AWS offer candidate columns for mapping to ListUnitPrice within billing data, relying on pricing data for all providers is advisable due to mentioned discrepancies and volume/tier-based pricing concerns. Filtering pricing data based on resolved billing data to identify that specific list unit price is possible but often presents a challenge, particularly with volume/tier-based prices.
+- MSFT lists the Retail Price as well as the Unit Price.
+
+### Prices and Currencies
+
+- We deliberated on whether to specify both BillingCurrency and PricingCurrency and decided to enforce a single currency, specifically BillingCurrency. This approach provides consistency and simplifies invoice reconciliation. Since some providers don't include List Unit Prices in BillingCurrency in their public price sheets, the inclusion of a CurrencyExchangeRate dimension in the billing data becomes imperative (name subject to change). This is necessary to ensure we can accurately compare and match ListUnitPrices provided in billing data with those published in public price sheets.
+
+### Naming challenges
+
+Two naming challenges were resolved through polls, giving signed members the chance to voice their preferences.
+
+- POLL 1: Please select the term you prefer to use when referring to the published rate and corresponding cost without any discounts:
+  - Retail - 3 votes
+  - List - 13 votes
+  - Market - 1 vote
+  - PAYG - 1 vote
+  - Public - 5 votes
+- POLL 2: Please select the term you prefer to use when referring to the price for a single unit of measure:
```

Comment:
A price refers to a fixed amount of money. A rate refers to a set amount of money which is charged according to some period of time or some number of items. The price is $50 vs. The rate is $50/hour.